terraform/groups/ecs-service/locals.tf

```diff
@@ -50,48 +42,42 @@ locals {
   staffware_db_user             = local.service_secrets["staffware_db_user"]
   staffware_db_password         = local.service_secrets["staffware_db_password"]
   staffware_db_connectionstring = local.service_secrets["staffware_db_connectionstring"]
-  mongodb_url                   = local.service_secrets["mongodb_url"]
 
   # create a map of secret name => secret arn to pass into ecs service module
   # using the trimprefix function to remove the prefixed path from the secret name
   secrets_arn_map = {
-    for sec in data.aws_ssm_parameter.secret:
-      trimprefix(sec.name, "/${local.name_prefix}/") => sec.arn
+    for sec in data.aws_ssm_parameter.secret :
+    trimprefix(sec.name, "/${local.name_prefix}/") => sec.arn
   }
 
   service_secrets_arn_map = {
-    for sec in module.secrets.secrets:
-      trimprefix(sec.name, "/${local.service_name}-${var.environment}/") => sec.arn
+    for sec in module.secrets.secrets :
+    trimprefix(sec.name, "/${local.service_name}-${var.environment}/") => sec.arn
```

Comment:
https://github.com/companieshouse/transaction-search-tool/pull/90/files#r1335548947

Reply:
Not really, updated automatically by the linter.